src/types/__tests__/profile.test.ts

```diff
@@ -50,6 +52,7 @@ const mockedExtendedProfile: ExtendedProfileApi = {
   is_test_profile: false,
   is_webhook_enabled: anIsWebhookEnabled,
   preferred_languages: aPreferredLanguages,
+  reminder_status: aReminderStatus,
```

Comment:
We should add this to the first `it` test checks, otherwise we aren't test the value.
```typescript
expect(userData.reminder_status).toBe(
  mockedExtendedProfile.reminder_status
);
```

Reply:
Should we add even `last_app_version`?